src/Framework/TargetBuiltReason.cs

```diff
@@ -28,6 +28,22 @@ public enum TargetBuiltReason
         /// <summary>
         /// The target was part of the parent's AfterTargets list.
         /// </summary>
-        AfterTargets
+        AfterTargets,
+
+        /// <summary>
+        /// The target was defined as an initial target of the project.
+        /// </summary>
+        InitialTarget,
+
```

Comment:
maybe remove the empty line. As a rule in C# there shouldn't be two consecutive empty lines.